### ISSUE TITLE
The demo is missing the links for Authentication

### ISSUE THREAD

**Issue by Exadra37** [pow-auth/pow_demo#3]
A demo without links to use the demo makes it less intuitive to use, because it means you need to know how Pow works, and normally demos are for onboard new users of the Library.

**Comment by danschultzer** [comment]
You're right. I'll be using this repo to ensure that the guides are working (easier to test changes with this already set up), so the master branch only have the first few steps from the readme done.

It makes sense to also have a full demo app too that you can spin up.

**Comment by Exadra37** [comment]
> It makes sense to also have a full demo app too that you can spin up.

As you know from the Elixirforum I have been looking for a Auth solution, and the lack of proper demos in your repos made me giving up in trying to test all your different repos, and in the end going with another package that as working demos that make a breeze to understand how everything fits together.

Despite I was in love by the Pow approach it's too much moving parts to get all working together for someone that is new to the programming language and is trying to understand the best Auth library to use for his use case.

Keep the good working in the Library and maybe try to work on some good working demos to see all Pow repos working together :)

**Comment by danschultzer** [comment]
Yeah, that's understandable. I've been thinking of taking some inspiration from the new `phx.gen.auth` generator and have one generator mix task to set up everything so you just need to start up the app with zero configuration.

**Comment by danschultzer** [comment]
Full app here btw: https://github.com/pow-auth/pow_demo/tree/full-app

**Comment by Exadra37** [comment]
Thanks for the link to the `full-app` branch :)

Two things:

First you may want to make clear in the demo that you must search the Logs for the confirmation email link ;)

Second you may want to fix this error after log in:

![Screenshot from 2020-04-21 23-38-48](https://user-images.githubusercontent.com/1745456/79921225-b5645b00-8429-11ea-830c-7fb73effd57f.png)

**Comment by danschultzer** [comment]
Oh, thanks! Fixed and updated to use Phoenix 1.5 now

I think I got a much better setup in the readme now: https://github.com/pow-auth/pow_demo#pow-demo-app

**Comment by danschultzer** [comment]
I've added a lot more guides. I will expand later on the full demo app when I've got more time, let me know if you see anything missing 😄

**Comment by Exadra37** [comment]
I just tries the branch `all-extensions` and got nothing to try:

![Screenshot from 2020-04-23 08-36-22](https://user-images.githubusercontent.com/1745456/80072107-ad95db00-853d-11ea-805b-54dc6f3f2b5d.png)

No registration or login functionality...

![Screenshot from 2020-04-23 08-38-39](https://user-images.githubusercontent.com/1745456/80072255-e5048780-853d-11ea-8d4f-a83fca3fd5a7.png)

**Comment by danschultzer** [comment]
Oh right, the readme wasn't too clear about that. Updated and rearranged https://github.com/pow-auth/pow_demo#pow-demo-app so it explicitly states that the `master` and `all-extensions` branches only contain the absolute minimum (the routes are still there but no links). Wdyt?

**Comment by Exadra37** [comment]
Ok it works nows, but just shows links to the register, login and logout.

If I was you I woulld remove all that boilerplate from Phoenix in the Home page and personalize it to your Pow project, and include links to trigger each of the routes.

Demos are crucial to be fully functional, otherwise they are just a waste of time for both the author and to anyone trying them. I don't want to seem harsh, but this is what I do in my work, and your product seems to have a lot of potential, but is lacking in easy on-boarding.

**Comment by danschultzer** [comment]
Yeah, I agree with you 100%. I'll work on the `full-app` demo to make it a realistic app with all the functionality you would expect in any production platform, including UI. Thanks for trying this out and the feedback!

**Comment by Exadra37** [comment]
Just remember to have `master` being the home for the full featured app, because that way it just works ;)
